examples/cra/package.json

```diff
@@ -3,13 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@cowprotocol/cow-sdk": "../../",
-    "@safe-global/api-kit": "^1.3.0",
-    "@safe-global/protocol-kit": "^1.2.0",
-    "@testing-library/jest-dom": "^5.16.5",
+    "@cowprotocol/cow-sdk": "5.10.2",
+    "@safe-global/api-kit": "^2.4.0",
+    "@safe-global/protocol-kit": "^5.0.0",
+    "@testing-library/jest-dom": "^6.1.6",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "ethers": "^5.7.2",
+    "ethers": "^6.0.0",
```

Comment:
mm, I wish it was so easy!

Reply:
So, what do we do?
The tests still pass, but we know it'll break stuff down the line.
Can we just not update ethers?

Reply:
I think there's an oper grant to do this. If we update there's we need to update in CoW Swap and adapt. 
Also, there's some dependency with the contracts. We should do it properly, and we have a grantee fully focusing on this so I will close this PR